src/clusterfuzz/_internal/system/process_handler.py

```diff
@@ -565,6 +565,16 @@ def terminate_stale_application_instances():
     # Artifical sleep to let the processes get terminated.
     time.sleep(1)
 
+  elif platform == 'FUCHSIA':
+    processes_to_kill += [
+        'undercoat',
+        llvm_symbolizer_filename,
+    ]
+    terminate_processes_matching_names(processes_to_kill, kill=True)
+
+    # Artifical sleep to let the processes get terminated.
+    time.sleep(1)
```

Comment:
We can probably skip this. This hack looks Windows-specific.

Reply:
Ok; removed!